staging/src/k8s.io/apimachinery/pkg/util/dump/dump.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+
+NOTE: the contents of this file has been copied from k8s.io/kubernetes/pkg/util/dump.
+The reason for duplicating this code is to remove dependencies to k8s.io/kubernetes
+in the staging repos. Once k8s.io/kubernetes/pkg/util/dump is moved to an external
+repository(kubernetes/utils), this file should be removed and replaced with that one.
+*/
+
+package dump
+
+import (
+	"github.com/davecgh/go-spew/spew"
+)
+
+var prettyPrintConfig = &spew.ConfigState{
```

Comment:
Do we really need a full copy of this file in apimachinery?  Is there any simpler way?  If we can settle on a format we like, maybe we move it all to k/utils?